tests/ui_tools/test_boxes.py

```diff
@@ -235,15 +235,39 @@ def test__compose_attributes_reset_for_private_compose(
         mocker.patch("urwid.connect_signal")
         write_box.model.user_id_email_dict = user_id_email_dict
         write_box.private_box_view(recipient_user_ids=[11])
+
+        # Testing input less than 15 characters; >15 prompts confirmation popup
         write_box.msg_write_box.edit_text = "random text"
 
         size = widget_size(write_box)
         write_box.keypress(size, key)
 
+        write_box.view.controller.exit_compose_confirmation_popup.assert_not_called()
         assert write_box.to_write_box is None
         assert write_box.msg_write_box.edit_text == ""
         assert write_box.compose_box_status == "closed"
 
+    @pytest.mark.parametrize("key", keys_for_command("GO_BACK"))
+    def test__compose_attributes_reset_for_private_compose_popup(
```

Comment:
It is useful to separate the function name from the outcome/action by an extra underscore, so here before the `popup`, so it doesn't read as part of the function name.

It would also be useful to rename the existing related tests to indicate they are no longer a 'plain' test but for a different expected outcome/action.

Reply:
Revised the function names; are they appropriate now?